bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerOpenOp.java

```diff
@@ -86,6 +88,7 @@ public LedgerOpenOp(BookKeeper bk, BookKeeperClientStats clientStats,
         this.enableDigestAutodetection = bk.getConf().getEnableDigestTypeAutodetection();
         this.suggestedDigestType = digestType;
         this.openOpLogger = clientStats.getOpenOpLogger();
+        this.scheduler = OrderedScheduler.newSchedulerBuilder().numThreads(1).name("LedgerOpenOp").build();
```

Comment:
No need to create a new pool.
As we know the ledgerId we can probably use the same thread used for all the operations on the ledger

Reply:
We cannot use executor in bk,  if bk is closed unexpectedly, the operation will never completed.

Reply:
we cannot start a new thread per each LederOpenOp

if the BK client closes all the pending operations must fail somehow. we should fix that
otherwise there will be other operations that will never complete

Reply:
"some unit test would stuck forever"

maybe  we are missing to mock the executor

Reply:
fixed. @eolivelli